static_src/components/service_instance.jsx

```diff
@@ -186,7 +185,8 @@ export default class ServiceInstance extends React.Component {
               { serviceInstance.name }
             </span>
           </span>
-          <span className={ this.styler('panel-column') }>
+          <span className={ this.styler('panel-column', 'panel-column-less',
```

Comment:
This was for the cost of the service aligning it to the right. Is there another class that should be used in this scenario?